lib/useKResponsiveWindow/index.js

```diff
@@ -1,133 +1,152 @@
-import { computed, onBeforeUnmount, onMounted, ref } from '@vue/composition-api';
+import { computed, onBeforeUnmount, onMounted, ref, watch } from '@vue/composition-api';
 import useKWindowDimensions from '../useKWindowDimensions';
 import MediaQuery from './MediaQuery';
 
+/** Global variables */
+const { windowWidth, windowHeight } = useKWindowDimensions();
+
+/*
+  Implementing breakpoint controlled by watchers to work around
+  optimization issue: https://github.com/vuejs/vue/issues/10344
+  If that issue ever gets addressed, we should make them computed props.
+*/
+
+// should this be ref (default = null? 0?) or undef?
+const windowBreakpoint = ref(null) // as in prev version of composable
+// const windowBreakpoint = undefined; // as in mixin version
+
+
+const windowIsPortrait = ref(false);
+const windowIsLandscape = ref(false);
+const windowGutter = ref(16);
+const windowIsShort = ref(false);
+
+// should these be .value or just plain const? (if windowBreakpoint is no longer a ref)
+const windowIsLarge = computed(() => windowBreakpoint.value > 2);
+const windowIsMedium = computed(() => windowBreakpoint.value === 2);
+const windowIsSmall = computed(() => windowBreakpoint.value < 2);
+
+const orientationQuery = new MediaQuery('screen and (orientation: portrait)', event => {
+  updateOrientation(event.matches);
+  updateGutter();
```

Comment:
I think you can actually remove this from here, as this is only dependent on the window dimensions, right, not the orientation?